core/src/main/scala/kafka/server/DelegationTokenManager.scala

```diff
@@ -227,56 +161,10 @@ class DelegationTokenManager(val config: KafkaConfig,
     tokenCache.removeCache(tokenId)
   }
 
-  /**
-   *
-   * @return
-   */
-  def expireTokens(): Unit = {
-    lock.synchronized {
-      for (tokenInfo <- getAllTokenInformation) {
-        val now = time.milliseconds
-        if (tokenInfo.maxTimestamp < now || tokenInfo.expiryTimestamp < now) {
-          info(s"Delegation token expired for token: ${tokenInfo.tokenId} for owner: ${tokenInfo.owner}")
-          removeToken(tokenInfo.tokenId)
```

Comment:
Could you please merge `protected def removeCache(tokenId: String)` into `def removeToken(tokenId: String)`?